src/core/caniuse.js

```diff
@@ -186,14 +187,16 @@ function createTableHTML(conf, stats) {
       .sort(semverCompare)
       .slice(-numVersions)
       .reverse();
-
     const { support, title } = getSupport(latestVersion);
+    const cssClass= `caniuse-cell ${support}`;
     return hyperHTML`
-      <ul class="caniuse-browser">
-        <li class="${"caniuse-cell " + support}" title="${title}">
+      <div class="caniuse-browser">
```

Comment:
Can you help me understand why we need this div?

Reply:
In order to make hover work properly (and avoid layout shift), we added `position: absolute` to the `ul`. But in order to show `ul` in it's browser area, we wrap it in `.caniuse-browser` which has `position: relative`.
I am not sure of some other way of doing this. Do you want me to look into it?

Reply:
It's fine. Thanks for clarifying.